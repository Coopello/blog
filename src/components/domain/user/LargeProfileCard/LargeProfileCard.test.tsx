import renderer from "react-test-renderer";

import { LargeProfileCard } from "./";

describe("domain/user/LargeProfileCard", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <LargeProfileCard
        imageUrl="https://placehold.jp/150x150.png"
        myUrl={{
          twitter: "https://twitter.com/akt_prs10",
          github: "https://github.com/akito-10",
        }}
        name="Akito Fukuda"
        description="Web Dev"
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
