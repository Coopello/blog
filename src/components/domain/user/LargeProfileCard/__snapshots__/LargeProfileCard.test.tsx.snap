// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`domain/user/LargeProfileCard Snap Shot 1`] = `
<div
  className="flex w-[790px] max-w-full items-center gap-12 rounded-xl border p-12"
>
  <span
    className="relative h-[100px] w-[100px]"
  >
    <img
      alt="プロフィールアイコン"
      className="rounded-full object-contain"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      sizes="100vw"
      src="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=3840&q=75"
      srcSet="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=3840&q=75 3840w"
      style={
        {
          "bottom": 0,
          "color": "transparent",
          "height": "100%",
          "left": 0,
          "objectFit": undefined,
          "objectPosition": undefined,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "100%",
        }
      }
    />
  </span>
  <div>
    <div
      className="flex items-center gap-4"
    >
      <p
        className="text-xl font-bold"
      >
        Akito Fukuda
      </p>
      <div
        className="flex"
      >
        <a
          className="undefined grid place-content-center rounded-full p-2 hover:bg-gray-200"
          href="https://twitter.com/akt_prs10"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span>
            <svg
              fill="none"
              height="16"
              viewBox="0 0 16 16"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5.03168 14.5005C11.0694 14.5005 14.3718 9.49823 14.3718 5.16031C14.3718 5.01823 14.3718 4.87679 14.3622 4.73599C15.0047 4.27129 15.5593 3.69591 16 3.03679C15.4009 3.30239 14.7654 3.47649 14.1146 3.55327C14.7999 3.14306 15.3128 2.49779 15.5578 1.73759C14.9134 2.11999 14.2084 2.38947 13.4733 2.53439C12.9783 2.0081 12.3237 1.65961 11.6108 1.54284C10.8978 1.42607 10.1663 1.54753 9.52931 1.88842C8.89234 2.22931 8.38548 2.77064 8.08716 3.42862C7.78884 4.0866 7.71569 4.82456 7.87904 5.52831C6.57393 5.46284 5.29717 5.12366 4.13164 4.53279C2.9661 3.94192 1.93784 3.11256 1.1136 2.09855C0.693819 2.82121 0.565248 3.6767 0.754066 4.49083C0.942885 5.30496 1.43489 6.01652 2.12992 6.48063C1.60749 6.46532 1.09643 6.32438 0.64 6.06975V6.11135C0.640207 6.86925 0.902567 7.60374 1.38258 8.19026C1.86259 8.77677 2.53071 9.17919 3.2736 9.32927C2.79032 9.46109 2.28325 9.48036 1.79136 9.38559C2.00121 10.0378 2.40962 10.6081 2.95949 11.0169C3.50937 11.4256 4.17322 11.6523 4.85824 11.6653C4.17763 12.2003 3.39821 12.5958 2.56458 12.8293C1.73096 13.0627 0.859476 13.1296 0 13.0259C1.50122 13.9893 3.24795 14.5003 5.03168 14.4979"
                fill="#222222"
              />
            </svg>
          </span>
        </a>
        <a
          className="undefined grid place-content-center rounded-full p-2 hover:bg-gray-200"
          href="https://github.com/akito-10"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span>
            <svg
              fill="none"
              height="16"
              viewBox="0 0 16 16"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clipPath="url(#clip0_248_1027)"
              >
                <path
                  clipRule="evenodd"
                  d="M8.00107 0.333374C6.10189 0.334359 4.26498 1.00701 2.81881 2.23106C1.37263 3.4551 0.411497 5.15071 0.107258 7.01471C-0.196981 8.8787 0.175513 10.7895 1.15814 12.4055C2.14077 14.0215 3.66945 15.2372 5.47085 15.8354C5.86826 15.9091 6.01796 15.6628 6.01796 15.4534C6.01796 15.2439 6.01001 14.6367 6.00736 13.9729C3.78183 14.4536 3.31155 13.0337 3.31155 13.0337C2.94858 12.1117 2.42399 11.8693 2.42399 11.8693C1.69804 11.3767 2.4783 11.3859 2.4783 11.3859C3.28241 11.4425 3.705 12.2065 3.705 12.2065C4.4177 13.4223 5.57682 13.0706 6.03253 12.8651C6.10406 12.3501 6.31205 11.9997 6.54123 11.8008C4.76345 11.6006 2.89559 10.9183 2.89559 7.87032C2.88457 7.07985 3.17958 6.31541 3.71957 5.73516C3.63744 5.53495 3.36321 4.7262 3.79772 3.62766C3.79772 3.62766 4.46936 3.41428 5.99809 4.443C7.30934 4.08646 8.69281 4.08646 10.0041 4.443C11.5315 3.41428 12.2018 3.62766 12.2018 3.62766C12.6376 4.72356 12.3634 5.53231 12.2813 5.73516C12.823 6.3155 13.1186 7.08127 13.1065 7.87296C13.1065 10.9275 11.2347 11.6006 9.45429 11.7969C9.74043 12.0445 9.9961 12.5279 9.9961 13.2708C9.9961 14.3351 9.98683 15.1913 9.98683 15.4534C9.98683 15.6654 10.1312 15.9131 10.5366 15.8354C12.3382 15.2371 13.867 14.0212 14.8496 12.4049C15.8323 10.7887 16.2046 8.87757 15.9 7.01341C15.5954 5.14924 14.6338 3.45361 13.1871 2.22978C11.7405 1.00594 9.90313 0.333735 8.00372 0.333374H8.00107Z"
                  fill="#222222"
                  fillRule="evenodd"
                />
                <path
                  d="M3.02905 11.7547C3.01183 11.7942 2.94825 11.806 2.89658 11.7784C2.84492 11.7507 2.8065 11.6993 2.82505 11.6585C2.8436 11.6177 2.90586 11.6071 2.95752 11.6348C3.00919 11.6625 3.04893 11.7152 3.02905 11.7547Z"
                  fill="#222222"
                />
                <path
                  d="M3.35545 12.1143C3.32802 12.128 3.29663 12.1319 3.26667 12.1252C3.2367 12.1184 3.21002 12.1015 3.19119 12.0774C3.13953 12.0221 3.12892 11.9457 3.16866 11.9115C3.20841 11.8772 3.27995 11.893 3.33161 11.9483C3.38327 12.0037 3.3952 12.0801 3.35545 12.1143Z"
                  fill="#222222"
                />
                <path
                  d="M3.6686 12.5714C3.61959 12.6056 3.53612 12.5714 3.48976 12.5029C3.47694 12.4906 3.46674 12.4758 3.45978 12.4595C3.45281 12.4433 3.44922 12.4257 3.44922 12.408C3.44922 12.3903 3.45281 12.3728 3.45978 12.3565C3.46674 12.3402 3.47694 12.3255 3.48976 12.3132C3.53877 12.2803 3.62224 12.3132 3.6686 12.3804C3.71497 12.4475 3.71629 12.5371 3.6686 12.5714V12.5714Z"
                  fill="#222222"
                />
                <path
                  d="M4.09578 13.0139C4.05207 13.0626 3.96331 13.0494 3.89045 12.9836C3.81759 12.9177 3.80037 12.8281 3.84408 12.7807C3.8878 12.7333 3.97656 12.7465 4.05206 12.811C4.12757 12.8756 4.14215 12.9665 4.09578 13.0139V13.0139Z"
                  fill="#222222"
                />
                <path
                  d="M4.69875 13.272C4.67888 13.3339 4.5888 13.3616 4.49872 13.3352C4.40864 13.3089 4.34902 13.2351 4.36625 13.1719C4.38347 13.1087 4.47487 13.0797 4.56628 13.1087C4.65768 13.1377 4.71597 13.2075 4.69875 13.272Z"
                  fill="#222222"
                />
                <path
                  d="M5.35091 13.3169C5.35091 13.3814 5.27673 13.4367 5.18135 13.438C5.08597 13.4394 5.00781 13.3867 5.00781 13.3221C5.00781 13.2576 5.08199 13.2023 5.17737 13.201C5.27275 13.1996 5.35091 13.251 5.35091 13.3169Z"
                  fill="#222222"
                />
                <path
                  d="M5.96289 13.2154C5.97481 13.28 5.90858 13.3472 5.8132 13.363C5.71782 13.3788 5.63436 13.3406 5.62244 13.2774C5.61051 13.2141 5.6794 13.1456 5.77213 13.1285C5.86486 13.1114 5.95097 13.1509 5.96289 13.2154Z"
                  fill="#222222"
                />
              </g>
              <defs>
                <clipPath
                  id="clip0_248_1027"
                >
                  <rect
                    fill="white"
                    height="16"
                    width="16"
                  />
                </clipPath>
              </defs>
            </svg>
          </span>
        </a>
      </div>
    </div>
    <p
      className="text-sm"
    >
      Web Dev
    </p>
  </div>
</div>
`;

exports[`domain/user/LargeProfileCard Snap Shot no myLinks links 1`] = `
<div
  className="flex w-[790px] max-w-full items-center gap-12 rounded-xl border p-12"
>
  <span
    className="relative h-[100px] w-[100px]"
  >
    <img
      alt="プロフィールアイコン"
      className="rounded-full object-contain"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      sizes="100vw"
      src="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=3840&q=75"
      srcSet="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F150x150.png&w=3840&q=75 3840w"
      style={
        {
          "bottom": 0,
          "color": "transparent",
          "height": "100%",
          "left": 0,
          "objectFit": undefined,
          "objectPosition": undefined,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "100%",
        }
      }
    />
  </span>
  <div>
    <div
      className="flex items-center gap-4"
    >
      <p
        className="text-xl font-bold"
      >
        Akito Fukuda
      </p>
      <div
        className="flex"
      />
    </div>
    <p
      className="text-sm"
    >
      Web Dev
    </p>
  </div>
</div>
`;
